core/main.py

```diff
@@ -25,6 +26,24 @@ def get_nearest_project_dir():
     return None
 
 
+def test_dbt_version():
+    # Test if the dbt version is compatible with this version of dbt-helper
+    installed_version = get_installed_version()
+
+    VERSION_INCOMPATIBILITY_MSG = """
+        Installed dbt version: {}
+        dbt-helper requires dbt version 0.17.X or higher.
+        You can find upgrade instructions here:
+        https://docs.getdbt.com/docs/installation
+    """.format(
+        installed_version.to_version_string(skip_matcher=True)
+    )
+
+    if int(get_installed_version().minor) < 17:
```

Comment:
This is going to break when dbt goes to `1.0.0` -- not sure the easiest way to compare symvers.

Reply:
I'd honestly want it to just break and force either of us to handle it then. I do foresee a lot more breaking changes as we move towards `1.0.0`. I think it might be best to just take it in strides for now, rather than baking in some jankier logic that lets `1.0.0` pass through in the future without being tested. I know there are some checks we could run where a new dbt version is released and it tests against dbt-helper to see if it runs. I've seen this in other projects but I have not set it up personally.

That might be the better solution and happy to open up an issue to have that built instead?

Reply:
Okay that works for me! Let's create that issue just so that we don't forget :)

Let's :shipit: -- are you down to try the deploy?

Reply:
Tracking here #36 

Happy to release! Mind giving it another review?